src/Tools/Source/CompilerGeneratorTools/Source/IOperationGeneratorNew/IOperationClassWriter.PortedTypes.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace IOperationGenerator
+{
+    internal sealed partial class IOperationClassWriter
+    {
+        private static readonly HashSet<string> PortedTypes = new HashSet<string>()
```

Comment:
Use an explicit string comparer here.

Reply:
Does this matter? It's not sticking around.